agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/localstorage/LocalFileSender.java

```diff
@@ -92,8 +94,11 @@ public void run() {
         resultCode.join(30, TimeUnit.SECONDS); // wait max 30 seconds for request to be completed.
       }
     } catch (RuntimeException ex) {
-      logger.error(
-          "Unexpected error occurred while sending telemetries from the local storage.", ex);
+      try (MDC.MDCCloseable ignored =
+          AzureMonitorMsgId.DISK_PERSISTENCE_LOADER_ERROR.makeActive()) {
```

Comment:
I often like to static import enums values where the context of the enum class name isn't needed (Intellij shortcut alt-enter on the enum value will bring up this option), might look nice throughout this PR

```suggestion
      try (MDC.MDCCloseable ignored = DISK_PERSISTENCE_LOADER_ERROR.makeActive()) {
```

Reply:
wish that can be included in the spotlessApply in the future? this will require me to go through every class to alt-enter

Reply:
will do though